p2p/module_raintree_test.go

```diff
@@ -228,58 +232,66 @@ func testRainTreeCalls(t *testing.T, origNode string, networkSimulationConfig Te
 		return iId < jId
 	})
 
+	prepareDNSResolverMock(t, valIds)
+
 	// Create connection and bus mocks along with a shared WaitGroup to track the number of expected
 	// reads and writes throughout the mocked local network
 	var wg sync.WaitGroup
-	connMocks := make(map[string]typesP2P.Transport)
-	count := 0
-	for _, valId := range valIds {
+	for i, valId := range valIds {
 		expectedCall := networkSimulationConfig[valId]
-		expectedReads := expectedCall.numNetworkReads + 1
+		expectedReads := expectedCall.numNetworkReads
```

Comment:
I'm not fresh on the details but can get them if you need. The gist of it was that the read count was incremented by one because each channel being used in the mock transports would emit once during setup or something like that.

Reply:
Makes a ton  of sense, thank you!